tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -64,7 +64,8 @@ def activate_edit_mode(self):
             release().\
             wait(1).\
             perform()
-        Wait(self.marionette).until(expected.element_displayed(app))
+        # why wait twice?  commented the next line; cause failure to occur.
+        # Wait(self.marionette).until(expected.element_displayed(app))
```

Comment:
If this wait is causing a failure you should remove/replace it instead of commenting it. Be sure to run all test that use the method to check that they are not affected.